static/app/views/alerts/rules/row.tsx

```diff
@@ -218,79 +241,17 @@ function RuleListRow({
       <ActionsRow>
         <Access access={['alerts:write']}>
           {({hasAccess}) => (
-            <React.Fragment>
-              <StyledDropdownLink>
-                <DropdownLink
-                  anchorRight
-                  caret={false}
-                  title={
-                    <Button
-                      tooltipProps={{
-                        containerDisplayMode: 'flex',
-                      }}
-                      size="small"
-                      type="button"
-                      aria-label={t('Show more')}
-                      icon={<IconEllipsis size="xs" />}
-                    />
-                  }
-                >
-                  <li>
-                    <Link to={editLink}>{t('Edit')}</Link>
-                  </li>
-                  <Confirm
-                    disabled={!hasAccess || !canEdit}
-                    message={tct(
-                      "Are you sure you want to delete [name]? You won't be able to view the history of this alert once it's deleted.",
-                      {
-                        name: rule.name,
-                      }
-                    )}
-                    header={t('Delete Alert Rule?')}
-                    priority="danger"
-                    confirmText={t('Delete Rule')}
-                    onConfirm={() => onDelete(slug, rule)}
-                  >
-                    <MenuItemActionLink title={t('Delete')}>
-                      {t('Delete')}
-                    </MenuItemActionLink>
-                  </Confirm>
-                </DropdownLink>
-              </StyledDropdownLink>
-
-              {/* Small screen actions */}
-              <StyledButtonBar gap={1}>
```

Comment:
![image](https://user-images.githubusercontent.com/1400464/152405171-8dbcca3b-b18f-4e58-84ba-c76ecebe127f.png)

the new dropdown doesn't seem to fix this issue with small screens. We used inline buttons here because our dropdown is appended to the current location instead of being appended to the body via popper or something

Reply:
oh good catch! The new dropdowns were misplaced because of some CSS positioning issues. I changed the table positioning and now it works as expected.

before:
<img width="541" alt="Screen Shot 2022-02-04 at 10 10 18 AM" src="https://user-images.githubusercontent.com/44172267/152583323-376e50bf-4e77-41ff-946e-f56b1e56d73f.png">

after:
<img width="540" alt="Screen Shot 2022-02-04 at 10 10 39 AM" src="https://user-images.githubusercontent.com/44172267/152583344-ed7b8759-6f3f-4715-a027-f548cc3c6010.png">
<img width="406" alt="Screen Shot 2022-02-04 at 10 33 22 AM" src="https://user-images.githubusercontent.com/44172267/152583947-9920789f-f43d-410c-ad00-b74a1e125457.png">

Reply:
nice!!! i've always hated those buttons